File: catalog/admin/section/ordering.php
<?php
defined('AUTH') or die('Restricted access');

include_once $_SERVER['DOCUMENT_ROOT'].'/components/catalog/admin/classes/AdminCatalogSection.php';
$SECTION = new AdminCatalogSection();

$arr['id'] = intval($SITE->d[5]);  // section id
$arr['type'] = $SITE->d[4];  // up / down

$catalog_id = $SECTION->getSection($arr['id'])['catalog_id'];

$SECTION->ordering($arr);

header('location: /admin/com/catalog/cat/'.$catalog_id);

File: dan/system/catalog/section/ordering.py
import sys
sys.path.append('system/catalog/classes')
from system.catalog.classes.Section import Section


def ordering(SITE):
    print('FUNCTION -> system-> calalog -> section -> ordering')
    data = {'id': SITE.p[3], 'type': SITE.p[2]}

    SECTION = Section(SITE)
    cat_id = SECTION.ordering(data)

    if 'cancel' in SITE.post:
        return {'redirect': '/system/catalog/cat/' + str(cat_id)}
    return {'redirect': '/system/catalog/cat/' + str(cat_id)}
